docs/content/how-to/ci/triggering-jobs-on-demand.md

```
@@ -0,0 +1,184 @@
+!!! tip ":bulb: Non-Claude Code editors"
+    The bundle can also be installed via APM for other editors:
+
+    ```bash
+    apm install openshift-eng/ai-helpers/plugins/openshift-developer --global --target cursor
+    ```
+
+### :key: Authenticate to the CI Cluster
+
+You need to be logged in to the `app.ci` cluster to use the Gangway API.
+
+1. Visit the [app.ci console ](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/)
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)